installation/vs-code-integration/new-project-wizard.md

```diff
@@ -20,12 +20,27 @@ To use the **Telerik UI for .NET MAUI Template Wizard**, install the `Telerik UI
 
 ## Start the Wizard
 
-You can create a Telerik .NET MAUI project by accessing the project template feature from the Visual Studio Code Extensions menu. To open the menu, use:
+Start the Telerik UI for .NET MAUI Template Wizard by using the following ways:
 
- - `Ctrl+Shift+P` on Windows/Linux 
- - `Cmd+Shift+P` on Mac. 
+* Press the **Telerik Extensions Information** icon in the Visual Studio Code activity bar.
 
-Next, select **Telerik UI for .NET MAUI Template Wizard: Launch** from the menu and press **Enter**. 
+1. Click on the **Telerik Extensions Information** icon
+1. Select **MAUI** tab
+1. Press **Create New Project**.
+1. The **Telerik UI for .NET MAUI Template Wizard** opens
+1. Using the Template Wizard you can create a new **Blank App** or **Blank Shell App**.
+
+![Telerik UI for .NET MAUI VS Code Extension](images/telerik-vs-code-extension.png)
+
+* From the Visual Studio Code Extensions menu
+
+1. To open the menu, use:
```

Comment:
Indent the ordered list items so that they get nested under the parent item. Indent the two unordered list items accordingly so that they don't interrupt the automatic numbering.